base/doc/clsguide.tex

```diff
@@ -977,6 +977,30 @@ \subsection{Passing options around}
 handler, but this handler is not used for |landscape| because that option is
 explicitly declared.
 
+\subsection{Testing dates of other code}
+
+\begin{decl}
+  |\IfPackageAtLeast(TF)| \arg{package} \arg{date} \arg{true code} \arg{false code}\\
+  |\IfClassAtLeast(TF)| \arg{class} \arg{date} \arg{true code} \arg{false code}\\
+  |\IfFileAtLeast(TF)| \arg{file} \arg{date} \arg{true code} \arg{false code}
+\end{decl}
+There are occasions when you need to test if a package is
+at least a particular date (assuming it has been loaded). This can be done using the
+commands |\IfPackageAtLeastTF|, etc. Separate commands are available to test
+packages, classes and general files: in all cases, they rely on the existence
+of a |\Provides...| line in the file being queried. Three versions
+of each command are available, ending |T|, |F| and |TF|, which indicate the
+outcome branches required. Thus for example |\IfPakcageAtLeastT| does not need
+a \meta{false code} argument. The \meta{date} argument can be given either in
+ISO (|YYYY-MM-DD|) or traditional \LaTeX{} (|YYYY/MM/DD|) formats.
+
+\begin{decl}
+  |\IfFormatAtLeast(TF)| \arg{date} \arg{true code} \arg{false code}
```

Comment:
but if I remember correctly everybody else was happy enough with it given that Expl3 is not possible and ExplIII or Expliii is horrible and the longer names do not do well either. My vote is on keeping the name.

Reply:
> I didn't include `\IfExplAtLeaseTF` since @car222222 is not happy with the name

Is he not?  I think that was in a somewhat different context. 
As @FrankMittelbach indicates, below, in this case the alternative is something like `\IfExplThreeAtLeaseTF`. 

It would be useful to explain that we use just "Expl" in places like this where a "3" is not permitted.